app/components/views/GetStartedPage/GetStarted.module.css

```
@@ -618,6 +618,16 @@
   max-width: 410px;
 }
 
+.releaseNotesText p,
+.releaseNotesText ul,
+.releaseNotesText ol,
+.releaseNotesText h1,
+.releaseNotesText h2,
+.releaseNotesText h3,
+.releaseNotesText h4 {
+  margin-bottom: 1em;
+}
+
```

To indent lists and ensure the bullets are displayed
```suggestion

.releaseNotesText ul,
.releaseNotesText ol {
  padding-left: 2em;
}
```

---

Just noticed this is likely fixed by #3033 3051

---

@ay-p I think this needs to be double-checked. It was committed into this PR, but this PR is not rebased onto #3051

The result could be a double-indentation on the lists.